src/todo.js

```diff
@@ -0,0 +1,7 @@
+export default class Todo {
+  constructor(description, index, completed = false) {
+    this.description = description;
+    this.index = index;
+    this.completed = completed;
+  }
+}
```

Comment:
- kindly make sure the functionality to make the task complete and clear all after making it complete is working as expected . on my side when I click  to clear all there is nothing which happens Kindly fix please fix that to make your website more usable  :+1:

Reply:
Hi @MaganaAsiati ,
The functionality to clear all completed tasks is a requirement for day four project. This is a pull request for day three and that is not among the requirements.
This pull request needs to be approved so that I can work on that feature.